coffee_lib/src/plugin.rs

```diff
@@ -125,19 +126,24 @@ impl Plugin {
     /// configure the plugin in order to work with cln.
     ///
     /// In case of success return the path of the executable.
-    pub async fn configure(&mut self, verbose: bool) -> Result<String, CoffeeError> {
+    pub async fn configure(
+        &mut self,
+        root_path: &str,
```

Comment:
```suggestion
```

Should be already inside the plugin, please take a look on how coffee index the plugins